release/pkg/manager/calico/manager.go

```diff
@@ -285,7 +289,7 @@ func (r *CalicoManager) Build() error {
 		env := append(os.Environ(), fmt.Sprintf("VERSION=%s", ver))
 		targets := []string{"release-windows-archive", "dist/install-calico-windows.ps1"}
 		for _, target := range targets {
-			if _, err = r.runner.RunInDir(r.repoRoot, "make", []string{"-C", "node", target}, env); err != nil {
+			if err = r.makeInDirectoryWithNoOutput(filepath.Join(r.repoRoot, "node"), target, env...); err != nil {
```

Comment:
Not a huge fan of all the places we're removing output - without any log output, it's impossible to tell what goes wrong if something fails.

Reply:
I've updated the fn to use `Run` instead of `RunNoCapture` as that was my original intention